projects/admin/src/app/record/detail-view/acq-order-detail-view/acq-order-detail-view.component.html

```diff
@@ -0,0 +1,91 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019 RERO
+  
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+  
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+  
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record$ | async as record">
+  <h1 class="mb-3">{{ record.metadata.order_number }}</h1>
+  <article>
+    <section class="m-2 p-2">
+      <!-- VENDOR -->
+      <dl class="row mb-0" *ngIf="record.metadata.vendor">
+        <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
```

Comment:
Is it possible to have something like:
```html
<dl>
  <dt>
  <dd>
  <dt>
  <dd>
</dl>
```

instead of:
```html
<dl>
  <dt>
  <dd>
<dl>
</dl>
  <dt>
  <dd>
</dl>
```

Reply:
ok